test/index.js

```diff
@@ -159,3 +159,29 @@ test('custom value encoding that retrieves a buffer from underlying store', func
 
   db.get('key', noop)
 })
+
+test('iterator options .keyAsBuffer and .valueAsBuffer defaults', function (t) {
+  t.plan(2)
+
+  var down = {
+    iterator: function (options) {
+      t.is(options.keyAsBuffer, true, '.keyAsBuffer defaults to true')
+      t.is(options.valueAsBuffer, true, '.valueAsBuffer defaults to true')
```

Comment:
I just noticed this is wrong since `utf8` is default encoding and should produce `buffer: false`